packages/integrations/react/README.md

```diff
@@ -61,6 +61,44 @@ To use your first React component in Astro, head to our [UI framework documentat
 - 💧 client-side hydration options, and
 - 🤝 opportunities to mix and nest frameworks together
 
+## Options
+
+### Children parsing
+
+When you pass children into a React component from an Astro component you only see a single child on the React side:
```

Comment:
```suggestion
Children passed into a React component from an Astro component are parsed as plain strings, not React nodes.

For example, the `<ReactComponent />` below will only receive a single child element:
```
I think we can tighten this explanation up? This might be a bit *too* terse? Make it accurate, but I think this kind of structure would work nicely!